inc/saiport.h

```diff
@@ -478,6 +486,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_ADVERTISED_FEC_MODE,
 
     /**
+     * @brief Advertised port FEC AN request bit.  True means FEC is requested for 10/40/100G
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST,
```

Comment:
In the current IEEE standard, that is true.  But 25G Consortium is different and more flexible.  Wanted to make attribute generic, not specific to the IEEE standard.  So if above is set for 25G for IEEE standard, it does nothing in the adapter.